### DISCUSSION TITLE
What part of index.html must we NOT touch

### DISCUSSION THREAD

**Discussion by assassin316** [unoplatform/uno#14386] [Q&A]
Uploading a wasm app works perfectly when we upload it as is (in my case in resides in bin/Release/net5.0/dist)

However when I attempt to modify the `index.html` file to change the meta info, the Title (actually, adding the Title tag doesn't work), adding Google Analytics scripts, etc, the app doesn't load:

![image](https://github.com/unoplatform/uno/assets/17514380/22e7e286-7360-49d9-9b5a-75f87914d34a)

I feel there are elements in the page that must be untouched. Is there any documentation on the required structure for `index.html `?

**Comment by jeromelaban** [comment]
Thanks for the question! The best way to change the index.html is to [take a look at this doc](https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/features-additional-files.html#indexhtml-content-override). You can provide your own, given that you keep the replacement tokens.

**Comment by assassin316** [comment]
Great!! 👍 Thank you, never saw this.
